### ISSUE TITLE
Error: "Class 'pocketmine\lang\Language' not found" (EXCEPTION) in "MagicWE2_dev-152.phar/src/xenialdan/MagicWE2/Loader" at line 69

### ISSUE THREAD

**Issue by teracube** [inxomnyaa/MagicWE#108]
issue with a new installation

https://poggit.pmmp.io/r/46727/MagicWE2_dev-152.phar
https://jenkins.pmmp.io/job/PocketMine-MP/1642/artifact/PocketMine-MP.phar

```
2018-11-30 [14:18:23] [Server thread/INFO]: Loading MagicWE2 v7.0.0
2018-11-30 [14:18:23] [Server thread/INFO]: Enabling MagicWE2 v7.0.0
2018-11-30 [14:18:23] [Server thread/CRITICAL]: Error: "Class 'pocketmine\lang\Language' not found" (EXCEPTION) in "MagicWE2_dev-152.phar/src/xenialdan/MagicWE2/Loader" at line 69
2018-11-30 [14:18:23] [Server thread/DEBUG]: #0 src/pocketmine/plugin/PluginBase(108): xenialdan\MagicWE2\Loader->onEnable()
2018-11-30 [14:18:23] [Server thread/DEBUG]: #1 src/pocketmine/plugin/PluginManager(568): pocketmine\plugin\PluginBase->setEnabled(boolean 1)
2018-11-30 [14:18:23] [Server thread/DEBUG]: #2 src/pocketmine/Server(1919): pocketmine\plugin\PluginManager->enablePlugin(xenialdan\MagicWE2\Loader object)
2018-11-30 [14:18:23] [Server thread/DEBUG]: #3 src/pocketmine/Server(1905): pocketmine\Server->enablePlugin(xenialdan\MagicWE2\Loader object)
2018-11-30 [14:18:23] [Server thread/DEBUG]: #4 PocketMine-DevTools_dev-114.phar/src/DevTools/DevTools(51): pocketmine\Server->enablePlugins(integer 0)
2018-11-30 [14:18:23] [Server thread/DEBUG]: #5 src/pocketmine/plugin/PluginBase(108): DevTools\DevTools->onEnable()
2018-11-30 [14:18:23] [Server thread/DEBUG]: #6 src/pocketmine/plugin/PluginManager(568): pocketmine\plugin\PluginBase->setEnabled(boolean 1)
2018-11-30 [14:18:23] [Server thread/DEBUG]: #7 src/pocketmine/Server(1919): pocketmine\plugin\PluginManager->enablePlugin(DevTools\DevTools object)
2018-11-30 [14:18:23] [Server thread/DEBUG]: #8 src/pocketmine/Server(1905): pocketmine\Server->enablePlugin(DevTools\DevTools object)
2018-11-30 [14:18:23] [Server thread/DEBUG]: #9 src/pocketmine/Server(1640): pocketmine\Server->enablePlugins(integer 0)
2018-11-30 [14:18:23] [Server thread/DEBUG]: #10 src/pocketmine/PocketMine(249): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/myserver/, string /home/myserver/plugins/)
2018-11-30 [14:18:23] [Server thread/DEBUG]: #11 /home/myserver/PocketMine-MP.phar(1): require(string phar:///home/myserver/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
2018-11-30 [14:18:23] [Server thread/INFO]: Disabling MagicWE2 v7.0.0
2018-11-30 [14:18:23] [Server thread/INFO]: [MWE2] Destroying Sessions
2018-11-30 [14:18:23] [Server thread/INFO]: [MWE2] Sessions successfully destroyed
```

**Comment by inxomnyaa** [comment]
This is due to 3.0.0 using BaseLang, but 4.0.0 using Language

https://github.com/thebigsmileXD/MagicWE2/commit/27a26167b90987396e8583b2e027e338e96a6f28
AKA "Beta-Update to 4.* developer version - **_DO NOT USE IN PRODUCTION_**"

And as of https://github.com/thebigsmileXD/MagicWE2/pull/100/commits/17c4601db63cd26bd9d29775bf4381d544373528 4.0.0 is not supported anymore.

**Comment by inxomnyaa** [comment]
Fixed by pull request #110
